Cargo.toml

```diff
@@ -16,20 +16,7 @@ opt-level = 3
 # lto = false
 
 [patch.crates-io]
-# If you need to temporarily test Servo with a local fork of some upstream
-# crate, add that here. Use the form:
-#
-#     <crate> = { path = "/path/to/local/checkout" }
-#
-# Or for a git dependency:
-#
-#     [patch."https://github.com/servo/<repository>"]
-#     <crate> = { path = "/path/to/local/checkout" }
-
-# This is here to dedupe winapi since mio 0.6 is still using winapi 0.2.
-mio = { git = "https://github.com/servo/mio.git", branch = "servo-mio-0.6.22" }
```

Comment:
Is this mio fork supposed to be removed here? Isn't it still required for using a compatible version of winapi?

Reply:
The newer versions of winit use a newer version of mio, so the fork can no longer be used. It was only necessary for UWP builds, and we're not shipping those in the Windows app store any more, so it's less important.

Reply:
mio 0.6 is still being used as a dependency of other crates though.